File: Kotlin/FizzBuzz.kt
//FizzBuzz coding interview question in Kotlin
//by Freya-The-Goddess

fun main() {
    val fizz: Int = 3
    val buzz: Int = 5
    val endNumber: Int = 100

    for (i in 1..endNumber) {
        var output = ""
        if (i % fizz == 0) output += "Fizz"
        if (i % buzz == 0) output += "Buzz"
        if (output.isEmpty()) output = i.toString()
        println(output)
    }
}

File: JavaScript/FizzBuzz.js
//FizzBuzz coding interview question in JavaScript
//by Freya-The-Goddess

function fizzbuzz() {
    var fizz = 3;
    var buzz = 5;
    var endNumber = 100;
    
    var result = ""
    for (let i = 1; i <= endNumber; i++) {
        output = ""
        output += (i % fizz == 0 ? "Fizz" : "")
        output += (i % buzz == 0 ? "Buzz" : "")
        result += (output || i.toString()) + "\r\n"
    }
    return result
}

function main() {
    document.getElementById("result").textContent = fizzbuzz();
}
